### PR TITLE
Added new screen reader strings

### PR DESCRIPTION
**Pull request by zainafzal08**

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -1,55 +1,55 @@
 {
   "appDesc": {
     "message": "A text editor for Chrome OS and Chrome.",
-    "description":"The description of the application, displayed in the web store."
+    "description": "The description of the application, displayed in the web store."
```
Comment (BugsNash) [inline _locales/en/messages.json]: optional: ideally these formatting consistency changes should be in a separate patch, so that the bmale history makes more sense

Comment (zainafzal08) [reply _locales/en/messages.json]: I'm actually so lost because these were already fixed in a previous CL

Comment (zainafzal08) [reply _locales/en/messages.json]: Turns out i needed a rebase :)
